.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+Use # to link to relevant PRs if the provided content is not part the present PR.
+
+- [ ] **changelog** provided, | not required because: 
+- [ ] **development PR** provided | not required because: 
```

Comment:
At other lines we have a `#` after the word `provided` to signal that we expect a PR number. Why don't we need that here?

Reply:
The idea is:

If the pr is a development pr
- [x] **development PR** provided | not required because: 

If the pr is a backport pr or a framework which needs to link to a dev pr
- [x] **development PR** provided # | not required because: 

But I agree it should be consistent so I will add the #